draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -909,7 +920,7 @@
    <section title="Expiry of Triggers" anchor="expiry-of-triggers">
    <t>
    dCDN MAY automatically delete trigger resources sometime after they reach a terminal state (one of "complete", 
-   "processed", "failed", or "cancelled"). In this case, after dCDN removed such trigger, it MUST respond 
+   "processed", "failed", or "cancelled"). In this case, after dCDN removed such a trigger, it MUST respond 
```

Comment:
It would be more correct to say, "after dCDN has removed such a trigger" (past perfect instead of past tense).